src/DynamoCoreWpf/Properties/Resources.en-US.resx

```diff
@@ -3229,4 +3229,13 @@ You can manage this in Preferences -&gt; Security.</value>
   <data name="TrustLocationSkippedNotification" xml:space="preserve">
     <value>No trusted location has been added.</value>
   </data>
+  <data name="MessagePackageTargetOtherHosts2" xml:space="preserve">
+    <value>The package or one of its dependencies targets a different environment, such as Revit, Civil 3D, Advance Steel, Alias or FormIt. This can cause instability and unexpected problems.</value>
```

Comment:
Can we be specific about which host specifically the package is targeting rather than saying it could be one of the many?

Reply:
Yes, I think I meant just the first host.